r2dbc-core/build.gradle

```diff
@@ -1,28 +1,25 @@
 plugins {
-    id "io.micronaut.build.internal.r2dbc-module"
+    id 'io.micronaut.build.internal.module'
```

Comment:
I would personally tend to keep the `r2dbc-module` plugin, even if it does nothing. The reason is that as soon as you will want to change something and that it needs to be done to all the modules, then you'll have to change the plugin applied to all modules, instead of just updating the convention plugin.

Reply:
I considered this, but as there's only a single module I decided to just remove the complexity